proto/proto/store/rpc.proto

```diff
@@ -122,7 +122,7 @@ message AccountProofsRequest {
     }
 
     // A list of account requests, including map keys + values.
-    repeated AccountRequest account_requests = 1;
+    AccountRequest account_request = 1;
```

Comment:
With this, the `code_commitments` field could change to only be be a single code commitment. The idea is that the user can send the last known code commitments of the accounts they were requesting, so as to minimize bandwidth by having the node not return the serialized `AccountCode` if the user was already storing it.

Since there would be one account per request now, there'd be no need to make it a list.